gql/mutations/user/loginMutation.ts

```diff
@@ -0,0 +1,42 @@
+export const loginMutation = `
+mutation login($loginInput:CustomerUserAuthInfoInput!) {
+  account:createCustomerAuthTicket(customerUserAuthInfoInput:$loginInput) {
+    customerAccount {
+      emailAddress
```

Comment:
Return same properties for customerAccount as the "getCurrentUser" query

Reply:
fixed.